jms/src/main/scala/akka/stream/alpakka/jms/javadsl/JmsSink.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.jms.javadsl
+
+import akka.NotUsed
+import akka.stream.alpakka.jms.{ JmsSinkSettings, JmsSinkStage }
+
+object JmsSink {
+
+  /**
+   * Java API: Creates an [[JmsSink]]
+   *
+   * @param jmsSinkSettings the jms connexion settings
```

Comment:
`connexion` -> `connection`, actually I think the type convey this information already, so skip the `@param` and `@return` annotations here, they do not  add any information that isn't already in the signature. Same for the other factory methods below.